assets/dfn-panel.css

```diff
@@ -0,0 +1,56 @@
+dfn {
```

Comment:
As this is being inserted into the document, we should add comment just explaining that this is the dfn-popup.